core/src/main/scala/com/fortysevendeg/mvessel/statement/StatementArguments.scala

```diff
@@ -19,7 +19,7 @@ class PreparedStatementArguments extends StatementArguments {
   private[this] var argumentsList: NonEmptyList[mutable.Map[Int, AnyRef]] = NonEmptyList(mutable.Map.empty)
 
   // Prepared statements count from 1
```

Comment:
The comment is fine, the prepared statements count from 1. 0 means no arguments.